### DISCUSSION TITLE
Copyrights, accessibility and translation of talks about Hedy

### DISCUSSION THREAD

**Discussion by ifsheldon** [hedyorg/hedy#3470] [Website]
Hi! I just watched your talk in Strange Loop. And I really like the idea of Hedy and appreciate the effort from every contributor!

Although I am a programmer, I don't have the capacity to technically get involved in this project. Instead, I would really like to help to get Hedy known more widely. Specifically, I'd like to:
* Translate your Strange Loop talk and the intro on the org website into Chinese
* Propose to have an alternative video source instead of YouTube since it's not accessible in China

But to do these, copyrights are of concern, so can you explain a bit on your policy/preference about that? Thank you!

**Comment by Felienne** [comment]
Hi @ifsheldon!!

Thanks for reaching out!

Translating the website is no problem, that I think is already translated entirely:

![image](https://user-images.githubusercontent.com/1003685/197752983-88166c6e-7b03-48cb-921c-563602f44362.png)

The question of the video is an interesting one.... I am totally ok with a different video source but I am not sure Strangeloop would allow us to download the video and place it elsewhere. I can't easily find a license on the video. 

A solution would be that you yourself record a video directly in Chinese! Maybe in collaboration with @Grace666-code? An Indonesian translator has also made their own videos about Hedy. Would that be an idea?

**Comment by ifsheldon** [comment]
> The question of the video is an interesting one....

I think the intro on the official website belongs to you, right? So I guess we can do the translation of the intro video first. As for the official website, I think we can just change the video source of its Chinese version to Bilibili while the sources of other versions stay on YouTube. But to upload videos to Bilibili, we need an account of it. If you can (or I can on behalf of Hedy) create an official Bilibili account, that should work out best.

> A solution would be that you yourself record a video directly in Chinese! Maybe in collaboration with @Grace666-code? An Indonesian translator has also made their own videos about Hedy. Would that be an idea?

Sure! I can do that.

**Reply by Felienne** [reply]
Yes the video we have now is made by me and can be moved, no problem! can you do that (I do not know Bilibili) once we have moved it, we can just update the link in Chinese only (in Weblate) Let me know if you need more help!

**Reply by ifsheldon** [reply]
OK! I will contact you on Discord if I see further issues.
